### ISSUE TITLE
sign up error : Unable to process request due to missing initial state

### ISSUE THREAD

**Issue by robdew** [warpdotdev/Warp#1109]
### Describe the bug

```text
trying to sign up with github, but I get an error that the request cannot be processed.  Persists across browser restarts, blocking extensions disabled.
```

### To Reproduce

Open warp app
click sign up
go to browser window (https://app.warp.dev/signup/remote?scheme=warp&public_beta=true)
click Sign up with Github

brief browser popup window appears, then disappears, then this message is displayed:

Unable to process request due to missing initial state. This may happen if browser sessionStorage is inaccessible or accidentally cleared.

Url (somewhat redacted) is: https://astral-field-294621.firebaseapp.com/__/auth/handler?.......
I am unsure what part of the URL is sensitive information.

### Expected behaviour

```text
Github authorization page.
```

### Operating System

MacOS

### OS Version

11.6.5

### Additional context

```text
It looks like there are similar, but not identical issues, related to Firebase logins.  I tried to make sure this wasn't a duplicate but may have missed an existing issue if it didn't include my exact error message.

I am using Firefox Multi-account containers but I have used them for years with Github authorizations/logins and never run into this error.

I can retest with another browser but that result doesn't matter to me because I can't use a one-off browser just to login to one app.
```

**Comment by dev4joel** [comment]
It seems like this issue.https://github.com/warpdotdev/Warp/issues/233.

**Comment by dev4joel** [comment]
btw,
How to Troubleshooting Login Issues is here :
- 👉 [Troubleshooting Login Issues](https://docs.warp.dev/help/troubleshooting-login-issues)

How to fix the confusing login flow issue is here:
- 👉 [Can Not Sign Up Or Sign In Warp With Github(Public Beta)](https://embiid.blog/post/Can-not-sign-up-or-sign-in-Warp-with-Github/)

How to fix the proxy issue is here:
- 👉 [WARP-does-not-work-after-submitting-an-invite-code](https://embiid.blog/post/WARP-does-not-work-after-submitting-an-invite-code/)

**Comment by vorporeal** [comment]
@robdew Thanks for the report!  Two questions:

1) Mind trying again today? We made a couple changes to this flow yesterday and it's possible they resolved your issue.
2) Any chance you're using a set of stricter security settings on Firefox that disable localStorage/the Web Storage APIs?  We use Firebase for authentication and it makes use of localStorage in its auth flow.

Also, you should only have to go through this login flow once, so if you test with a different browser and it works, you shouldn't have to use that browser for login again in the future.

**Comment by robdew** [comment]
Same error message today.

Firefox is configured to allow, but ask me, to use persistent storage. 

Here are the additional tests I added to my firefox login:

I signed out of github, because this allowed the ```github.com/login?client_id...``` popup to remain onscreen and I could see what it was asking. Before, it just popped up and disappeared because I was already signed in.

I went to ```https://app.warp.dev/signup/remote?scheme=warp&public_beta=true``` directly rather than through the Warp app.

 I signed in to github successfully, then the same error message appeared.

I switched to Chrome and it works OK, so I think it is safe to write this off as a strange interaction with Firefox multiaccount containers and Firebase (since other Github auth apps work fine).  I am used to the occasional quirk like this, but it rarely can't be resolved by manually changing the login workflow like I tried above.
